### DISCUSSION TITLE
Is any of the cuda_runtime_api simulated/emulated/invoked in trace-mode with AccelSim

### DISCUSSION THREAD

**Discussion by Yfeng-44** [accel-sim/accel-sim-framework#228] [Q&A] [answered]
Hi folks,

AccelSim beginner here, please bare me if my question looks stupid. 

I'm wondering are
1) any of the cuda_runtime_api simulated/emulated/invoked in trace-mode with AccelSim (it looks like it is not though)? 
2) or if I would like to change some behaviours of the emulated runtime, is it possible to properly do it with AccelSim?

Yuan

**Comment by JRPan** [comment, marked as answer]
1. No. CUDA runtime is not simulated in trace mode. 
2. Probably not. But if you can be more specific I can try to help

**Reply by Yfeng-44** [reply]
Hi JRPan,

Thanks so much for answering. I dont have a clear idea of what I should do with the runtime either as of now. Will re-open the thread when I have further thoughts.

thanks,
Yuan
